frontend/src/libs/components/modal/steps/delete-account/delete-account-message.tsx

```diff
@@ -0,0 +1,34 @@
+import { type FC } from 'react';
+
+import styles from './styles.module.scss';
+
+type DeleteAccountMessageProperties = {
```

Comment:
As far as I noticed, in the project we use just 
```suggestion
type Properties = {
```